src/App.vue

```diff
@@ -240,7 +234,8 @@ export default class App extends Mixins(
   // root store references
   @Getter(useRootStore) getKeycloakRoles!: Array<string>
   @Getter(useRootStore) isBootstrapFiling!: boolean
-  @Getter(useRootStore) isBootstrapTask!: boolean
+  @Getter(useRootStore) isBootstrapPending!: boolean
+  @Getter(useRootStore) isBootstrapTodo!: boolean
```

Comment:
There are now 3 getters for the bootstrap filings:
- does it belong in Filing History List?
- does it belong in Pending List? (still WIP)
- does it belong in Todo List?